pkg/hintrunner/zero/zerohint_signature.go

```diff
@@ -78,7 +68,7 @@ func newVerifyZeroHint(val, q hinter.ResOperander) hinter.Hinter {
 			if err != nil {
 				return err
 			}
-			qMv := memory.MemoryValueFromFieldElement(qFelt)
+			qMv := mem.MemoryValueFromFieldElement(qFelt)
```

Comment:
why not using `memory` as before?

Reply:
because the `MemoryValueFromFieldElement` was being referenced through the `memory` variable and for some reason throwing an error that `MemoryValueFromFieldElement` could not be found in the memory package. So i thought being 'explicit' on the package in which `MemoryValueFromFieldElement` is defined would do the trick, and it did!